content/docs/partnerships.mdx

```diff
@@ -164,7 +164,7 @@ Partners who successfully complete the Terraform Cloud Integration Checklist obt
 
 - Note: Currently, pre-apply is the only integration phase available at this time. As of September 2021, run tasks are available only as a beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization since this is currently enabled by default for our business tier customers of Terraform Cloud. If you have a customer that is interested in run tasks and are not a current Terraform Cloud for Business customer, customers can [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
 
-![TFC Badge](/img/docs/tfc-badge.png)
+<img src="/img/docs/tfc-badge.png" style={{display:'block', margin:'auto', width:'200px', height:'200px'}}/>
```

Comment:
Can you please add the `alt` attribute? Thank you!

Reply:
ahh - I'll remember to ask for this in future reviews - thank you Dylan!

Reply:
Thanks for the callout @dstaley, but what is the `alt`  attribute and where should it be added?